vep-mapping-pipeline/vep_mapping_pipeline/test/output_mappings.tsv

```diff
@@ -1723,7 +1722,7 @@
 7:151560613:T:C	1	ENSG00000106617	PRKAG2	missense_variant	0
 7:151781257:T:C	1	ENSG00000106617	PRKAG2	missense_variant	0
 7:151876578:A:G	1	ENSG00000106617	PRKAG2	missense_variant	0
-7:158918831:G:A	1	ENSG00000126870	WDR60	missense_variant	0
+7:158918831:G:A	1	ENSG00000126870	DYNC2I1	missense_variant	0
```

Comment:
@M-casado @apriltuesday This one looks worth evaluating in the genome browser (was the gene renamed? or are there two genes in that location?)

Reply:
It seems both are the same gene (share identifiers, chromosome coordinates, regions...) but **[WDR60](http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000126870;r=7:158649269-158749438)** is its gene name for *GRCh37*, while **[DYNC2I1](https://www.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000126870;r=7:158856558-158956747)** is its gene name for *GRCh38*. 

I would say the change within the ``output_mappings.tsv`` makes sense.

Reply:
Yes, definitely makes sense, thanks for checking!